env.go

```diff
@@ -122,6 +122,9 @@ type Options struct {
 	// TagName specifies another tag name to use rather than the default 'env'.
 	TagName string
 
+	// DefaultTagName specifies another default tag name to use rather than the default 'envDefault'.
+	DefaultTagName string
```

Comment:
Maybe this, because it's the tag about the default value

```suggestion
	DefaultValueTagName string
```

Reply:
Sounds good. Will make a change